variables.tf

```diff
@@ -38,6 +38,10 @@ variable "region" {
     condition     = contains(["eu-de", "us-south", "eu-gb", "jp-tok", "au-syd"], var.region)
     error_message = "You must specify 'eu-de', 'eu-gb', 'jp-tok', 'au-syd' or 'us-south' as the IBM Cloud region."
   }
+  validation {
+    condition     = local.kms_region == null || (local.kms_region == var.region)
+    error_message = "In case of enterprise plan, KMS instance should be in the same region as watsonx.data"
+  }
```

Comment:
Is it true for watsonx.data too? did we test?

Reply:
Your observation is correct. This is not the case in `watsonx.data`. I used KP instance created in `us-south` and `watsonx.data` instance using the Keys in `jp-tok` region. This worked. I will remove this validation.